src/main/java/nl/tudelft/ewi/devhub/server/web/resources/repository/AbstractProjectIssueResource.java

```diff
@@ -0,0 +1,200 @@
+		List<Issue> closedIssues = issues.findClosedIssues(repositoryEntity);
+
+		Map<String, Object> parameters = getBaseParameters();
+		
+		parameters.put("repository", repository);
+		parameters.put("openIssues", openIssues);
+		parameters.put("closedIssues", closedIssues);
+
+		List<Locale> locales = Collections.list(request.getLocales());
+		return display(templateEngine.process("courses/assignments/group-issues.ftl", locales, parameters));
+	
+	}
+	
+	@GET
+	@Transactional
+	@Path("/issues/create")
+	public Response openCreateIssuePage(@Context HttpServletRequest request) throws IOException{
+		
+		RepositoryEntity repositoryEntity = getRepositoryEntity();
+		RepositoryApi repositoryApi = getRepositoryApi(repositoryEntity);
+		RepositoryModel repository = repositoryApi.getRepositoryModel();
+		
+		Map<String, Object> parameters = getBaseParameters();		
+		parameters.put("repository", repository);
+		
+		List<Locale> locales = Collections.list(request.getLocales());
+		return display(templateEngine.process("courses/assignments/group-issue-create.ftl", locales, parameters));
+	}
+
+	@POST
+	@Transactional
+	@Path("/issues/create")
+	public Response createIssue(@Context HttpServletRequest request,
+			@FormParam("title") String title,
+			@FormParam("description") String description,
+			@FormParam("assignee") String assigneeNetID) throws IOException{
+		
+		Issue issue = new Issue();
+		
+		issue.setTitle(title);
+		issue.setDescription(description);
+		issue.setOpen(true);
+		
+		User assignee = users.findByNetId(assigneeNetID);
+		checkCollaborator(assignee);
+		
+		issue.setAssignee(assignee);
+		issue.setTimestamp(new Date());
+		issue.setRepository(getRepositoryEntity());
+		
+		issueBackend.createIssue(getRepositoryApi(getRepositoryEntity()), issue);
+		
+		return redirect(issue.getURI().toString());
+	}
+	@GET
+	@Transactional
+	@Path("/issue/{issueId}")
+	public Response viewIssue(@Context HttpServletRequest request, 
+			@PathParam("issueId") long issueId) throws IOException {
+
+		RepositoryEntity repositoryEntity = getRepositoryEntity();
+		RepositoryApi repositoryApi = getRepositoryApi(repositoryEntity);
+		RepositoryModel repository = repositoryApi.getRepositoryModel();
+		
+		Issue issue = issues.findIssueById(getRepositoryEntity(), issueId).get(0);
+		
+		Map<String, Object> parameters = getBaseParameters();		
+		parameters.put("repository", repository);		
+		parameters.put("issue", issue);
+		
+		List<Locale> locales = Collections.list(request.getLocales());
+		return display(templateEngine.process("courses/assignments/group-issue-create.ftl", locales, parameters));
+	}
+
+	@POST
+	@Transactional
+	@Path("/issue/{issueId}")
+	public Response updateIssue(@Context HttpServletRequest request, 
+			@PathParam("issueId") long issueId,
+			@FormParam("title") String title,
+			@FormParam("description") String description,
+			@FormParam("assignee") String assigneeNetID,
+			@FormParam("status") String status) throws IOException {
```

Comment:
I understand your point, but would you want is implemented in `AbstractIssue`, or just in the test? Since changing it in `AbstractIssue` would also have impact on `PullRequest`s....